src/database/types.ts

```diff
@@ -195,30 +196,18 @@ export type CreateArgs = {
 
 type Create = (args: CreateArgs) => Promise<Document>
 
-type UpdateArgs = {
-  collection: string
-  data: Record<string, unknown>
-  where: Where
-  draft?: boolean
-  locale?: string
-}
-
-type Update = (args: UpdateArgs) => Promise<Document>
-
-type UpdateOneArgs = {
+export type UpdateOneArgs = {
   collection: string,
   data: Record<string, unknown>,
-  where: Where,
-  draft?: boolean
+  id: string,
```

Comment:
If UpdateOne is only going to work by `id` then it should probably be renamed. We should discuss what is best here and for `deleteOne`.

Reply:
I'd like `UpdateByID` and `DeleteByID`. That way, we'd also be consistent with the local API.

Reply:
Just commenting here for posterity, the decision by the team this AM was to stick with `updateOne` and `deleteOne`. 👍